skimage/feature/tests/test_fisher_vector.py

```diff
@@ -0,0 +1,205 @@
+import pytest
+
+import numpy as np
+
+pytest.importorskip('sklearn')
+
+from skimage.feature.fisher_vector import (
+    learn_gmm, fisher_vector, FisherVectorException,
+    DescriptorException
+)
+
+
+def test_gmm_wrong_descriptor_format_1():
+    """
+    Test that DescriptorException is raised when wrong type for descriptions
```

Comment:
```suggestion
    """Test that DescriptorException is raised when wrong type for descriptions
```
Reference: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Done